crypto/engine/eng_linux_getrandom.c

```diff
@@ -0,0 +1,150 @@
+#include <openssl/opensslconf.h>
+
```

Comment:
On the contrary, I think your opinion is greatly respected here! @richsalz do you agree with closing this?